Cart.DataAccess/Models/ProductItemDal.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Cart.DataAccess.Models
+{
+    public class ProductItemDal
+    {
+        public int Id { get; set; }
```

Comment:
Use `[init](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/init)` setters if the setter is used just to initalize the property

```suggestion
        public int Id { get; init; }
```